MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -518,6 +518,14 @@ - (void)testDisableMachExceptionWorks {
 }
 #endif
 
+#if TARGET_OS_TV
```

Comment:
You can change the order with previous test method and do 
`#if TARGET_OS_TV`
`#else`
`#endif`